src/ExpressiveInstaller/config.php

```diff
@@ -11,13 +11,22 @@
         'zendframework/zend-expressive-twigrenderer'     => '^1.2.1',
         'zendframework/zend-expressive-zendrouter'       => '^2.0',
         'zendframework/zend-expressive-zendviewrenderer' => '^1.2.1',
-        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.0',
+        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.2',
     ],
 
     'require-dev' => [
         'filp/whoops'
     ],
 
+    'application' => [
+        'flat' => [
+            '/Resources/src/ConfigProvider.flat.php' => '/src/App/ConfigProvider.php',
```

Comment:
@Ocramius They're relative to the project root. By having the leading slash, it simplifies logic internally (we don't need to worry about concatenating a slash to the installer/project roots and/or between the installer/project roots and these paths). I'm willing to change them, but it risks adding complexity.